mesa/cookiecutter-mesa/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+All notable changes to this project will be documented in this file.
```

Comment:
The change log is the same as the history file, just a different way of calling it. I am thinking this might be a changelog for the model if the cookiecutter template is for the model. Does it have to be changelog or can it be history? (It doesn't matter to me what it is as long as it is consistent with the other. So, if we do change log here, we should probably rename the history file to change log as well. Make sense?